azure-kusto-data/Cargo.toml

```diff
@@ -12,40 +12,40 @@ keywords = ["sdk", "azure", "kusto", "azure-data-explorer"]
 categories = ["api-bindings"]
 
 [dependencies]
-arrow = { version = "13", optional = true }
-azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", rev = "66db4b485ce56b68be148708d9c810960a50be51", features = [
+arrow = { version = "15.0.0", optional = true }
+azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", rev = "8586a66b20fba463c39156f0390e583ec305ab2d", features = [
```

Comment:
I think we can consider moving to the released crates by now, by my impression especially core is pretty stable by now, and since last release the errors have been migrated in all crates. From my end that was one of the major milestones.

Reply:
Great! I changed it to concrete versions.